test_runner/regress/test_basebackup_error.py

```diff
@@ -1,23 +0,0 @@
-from __future__ import annotations
-
-from typing import TYPE_CHECKING
-
-import pytest
-
-if TYPE_CHECKING:
-    from fixtures.neon_fixtures import NeonEnv
-
-
-#
-# Test error handling, if the 'basebackup' command fails in the middle
-# of building the tar archive.
-#
-def test_basebackup_error(neon_simple_env: NeonEnv):
```

Comment:
Please let's not delete tests as part of hackathon.

Reply:
I renamed the `test_basebackup_error.py` file to `test_basebackup.py` to not create too many files, but git didn't acknowledge the renaming

The test is still there

Reply:
Gotcha!

Reply:
I moved it back for now to make it more isolated :)